### DISCUSSION TITLE
disable Node Runtime Detection Notification?

### DISCUSSION THREAD

**Discussion by Jeff-Lowrey** [mtxr/vscode-sqltools#1096] [Q&A] [answered]
Is there some way that I can prevent the SQLLiteDriver and/or SQLTools from showing a notification that the node runtime is discovered?

I've been looking through settings and I'm not finding anything,, but maybe I'm missing it?

**Comment by gjsjohnmurray** [comment, marked as answer]
Please try the `sqltools.disableNodeDetectNotifications` setting, which PR #1060 added.
